src/content/en/fundamentals/media/manipulating/files.md

```diff
@@ -0,0 +1,329 @@
+can take multiple stream descriptors. This means that Shaka Packager lets me
+split the audio and video in a single command.
+
+    packager \
+      input=myvideo.mp4,stream=video,output=myvideo_video.mp4 \
+      input=myvideo.mp4,stream=audio,output=myvideo_audio.m4a
+
+A full discussion of audio and video formats is beyond the scope of this
+article. One thing to note is that the audio stream of an mp4 file is the
+[advanced audio coding]([https://en.wikipedia.org/wiki/Advanced_Audio_Coding)
+format for which m4a is a common file extension.
+
+Shaka Packager presents demuxing as though you're _extracting_ a stream into a
+new file. It's a little different in ffmpeg, which presents as though you're
+_stripping_ the stream you don't want. With ffmpeg, you need two operations.
+
+    ffmpeg -i myvideo.webm -vcodec copy -an myvideo_video.webm
+    ffmpeg -i myvideo.webm -acodec copy -vn myvideo_audio.webm
+
+Just as with Shaka Packager, we have both an input and an output file. Another
+difference from Shaka Packager is that the streams are identified with flags
+that refer their codecs. The `-vcodec copy` and `-acodec copy` portions of the
+command tell ffmpeg to copy the streams I want while the `-an` and `-vn` flags
+strip the streams I don't want. The keyword copy means I'm moving the streams
+without changing their codecs. 
+
+A list of the
+[demuxing commands](/web/fundamentals/media/manipulating/cheatsheet#demux_split_audio_and_video)
+are provided on the cheat sheet.
+
+## Change the codec
+
+Continuing downward, we arrive at the codec. As stated earlier, a codec is _not_
+the same thing as a container. Two files of the same container type could hold
+data compressed using completely different codecs. The webm format for example
+allows audio to be encoded using either [vorbis](https://en.wikipedia.org/wiki/Vorbis)
+or [opus](https://en.wikipedia.org/wiki/Opus_(audio_format)). To change the
+codec I need ffmpeg.
+
+In the last section I demuxed the audio and video like this:
+
+    ffmpeg -i myvideo.webm -vcodec copy -an myvideo_video.webm
+    ffmpeg -i myvideo.webm -acodec copy -vn myvideo_audio.webm
+
+If I need to change the audio and video codec, I would replace the `copy` keyword
+with the name of a codec. For example, this command outputs an audio file
+encoded with the aac codec.
+
+    ffmpeg -i myvideo.mp4 -vn -c:a aac myvideo.m4a 
+
+The [cheat sheet](/web/fundamentals/media/manipulating/cheatsheet#codec)
+contains a short list of codecs used for DASH and HLS, and commands needed to
+convert to them.
+
+## File properties
+
+I've unpealed the onion. Yet if you compare what I've done to the list of stated
+goals you'll see that I'm not quite done. Anong the remaining items are bitrate
+and resolution. These properties correlate to the amount of data in a media
+file. It probably goes without saying, but I'm going to say it anyway, that you
+can always lower bitrate and resolution, but you can't increase them. (Well,
```

Comment:
You can increase both (e.g. SD to HD) and there are algorithms to help with quality. Remove that bit please.